pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -289,22 +289,24 @@ class Build {
                         context.jobDsl targets: templatePath, ignoreExisting: false, additionalParameters: jobParams
                     }
                 }
-               context.catchError {
-                    context.build job: jobName,
-                            propagate: false,
-                            parameters: [
-                                    context.string(name: 'UPSTREAM_JOB_NUMBER', value: "${env.BUILD_NUMBER}"),
-                                    context.string(name: 'UPSTREAM_JOB_NAME', value: "${env.JOB_NAME}"),
-                                    context.string(name: 'RELEASE_TAG', value: "${buildConfig.SCM_REF}"),
-                                    context.string(name: 'LABEL_ADDITION', value: additionalTestLabel),
-                                    context.string(name: 'KEEP_REPORTDIR', value: "${buildConfig.KEEP_TEST_REPORTDIR}"),
-                                    context.string(name: 'ACTIVE_NODE_TIMEOUT', value: "${buildConfig.ACTIVE_NODE_TIMEOUT}")]
+                def smoketest = context.build job: jobName,
+                                    propagate: true,
+                                    parameters: [
+                                        context.string(name: 'UPSTREAM_JOB_NUMBER', value: "${env.BUILD_NUMBER}"),
+                                        context.string(name: 'UPSTREAM_JOB_NAME', value: "${env.JOB_NAME}"),
+                                        context.string(name: 'RELEASE_TAG', value: "${buildConfig.SCM_REF}"),
+                                        context.string(name: 'LABEL_ADDITION', value: additionalTestLabel),
+                                        context.string(name: 'KEEP_REPORTDIR', value: "${buildConfig.KEEP_TEST_REPORTDIR}"),
+                                        context.string(name: 'ACTIVE_NODE_TIMEOUT', value: "${buildConfig.ACTIVE_NODE_TIMEOUT}")]
+                if ( smoketest.result == 'UNSTABLE' || smoketest.result == 'FAILURE' || smoketest.result == 'ABORTED')) {
```

Comment:
You could use the function "is worse than (SUCCESS)". And probably return it directly or ! negate it rather than storing a variable for it.
https://javadoc.jenkins-ci.org/hudson/model/Result.html#isWorseThan-hudson.model.Result-